assets/components/directDebit/directDebitActions.js

```diff
@@ -67,13 +67,20 @@ function payDirectDebitClicked(callback: Function): Function {
     }
 
     checkAccount(bankSortCode, bankAccountNumber, isTestUser, csrf)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error('code1');
+        }
+        return response.json();
+      })
       .then((response) => {
```

Comment:
You could perhaps name this something like `responseJson`, to distinguish it from the Response object above?

Reply:
In a separated PR later today, I will polish this function to add extra cases and I will see in that moment if it is better to have one or two functions and their variable names.